ingest-app/src/cmr/ingest/data/bulk_update.clj

```diff
@@ -128,7 +128,7 @@
        (let [statement (str "UPDATE bulk_update_coll_status "
                             "SET status = ?, status_message = ?"
                             "WHERE task_id = ? AND concept_id = ?")
-             status-message (util/trunc status-message 255)]
+             status-message (util/trunc status-message 4000)]
```

Comment:
I'm wondering if we'll have a byte vs. character issue. Some UTF-8 chars can take 2 bytes and so util/trunc needs to be consistent with the way Oracle works.

What I think can happen is that we'll truncate to 4000 chars which ends up being 4008 bytes and get an error trying to save in Oracle.

Reply:
You are correct, tested locally, larger UTF-8 chars will cause the update to fail if there are more bytes than allocated in the column.  This was true with 255 as well.

Reply:
I would just throw in a comment for now that with special UTF-8 characters that take 2 bytes and a status message larger than 4000 characters we will have an error updating.

We do not expect that we will have status messages this large though, and so it is not worth the effort to figure out how to truncate to 4000 bytes (4000 characters is fine). If we find problems due to this we will open a new ticket and address.

Reply:
I'm cool with that, then :-)